tidy3d/components/device/simulation.py

```
@@ -379,11 +479,34 @@ def plot_heat_conductivity(
             bounds=self.simulation_bounds, x=x, y=y, z=z, hlim=hlim, vlim=vlim
         )
 
-        cbar_cond = colorbar == "conductivity"
+        cbar_cond = True
 
-        ax = self.scene.plot_heat_conductivity(
-            ax=ax, x=x, y=y, z=z, cbar=cbar_cond, alpha=alpha, hlim=hlim, vlim=vlim
-        )
+        simulation_types = self._get_simulation_types()
+        if property is None:
+            if len(simulation_types) == 1:
+                if DeviceSimulationType.CONDUCTION in simulation_types:
+                    property = "electric_conductivity"
+                elif DeviceSimulationType.HEAT in simulation_types:
+                    property = "heat_conductivity"
+            else:
+                log.error(
```

let's raise a `ValueError` instead of using `log`